app/models/__init__.py

```diff
@@ -0,0 +1,20 @@
+from .access import *
+from .acl import ACL, Role
+from .base import *
+from .cron_job_run import CronJobRun
+from .join import JoinModel, join_model
+from .session import *
+from .token import *
+from .user import User
+from .util import *
```

Comment:
Why not use a consistent pattern here -- either import each item explicitly, or always use `*`?